InvoiceMaker/InvoiceMaker/Models/Repositories/ClientRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using InvoiceMaker.Data;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data.Entity;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.Models.Repositories
+{
+    public class ClientRepository
```

Comment:
For consistency, rename this class to `ClientRepo` or rename all of your other repositories from `*Repo` to `*Repository`.